runtime-testsuite/test/org/antlr/v4/test/runtime/go/BaseTest.java

```diff
@@ -424,15 +428,21 @@ public String execModule(String fileName) {
 	private String locateTool(String tool) {
 		ArrayList<String> paths = new ArrayList<String>(); // default cap is about right
 
-		String pathEnv = System.getenv("PATH");
-		if (pathEnv != null) {
-			paths.addAll(Arrays.asList(pathEnv.split(pathSep)));
-		}
+		// GOROOT should have priority if set
 		String goroot = System.getenv("GOROOT");
 		if (goroot != null) {
 			paths.add(goroot + File.separatorChar + "bin");
 		}
 
+		String pathEnv = System.getenv("PATH");
+		if (pathEnv != null) {
+			paths.addAll(Arrays.asList(pathEnv.split(pathSep)));
+		}
+
+		// OS specific default locations of binary dist as last resort
+		paths.add("/usr/local/go/bin");
+		paths.add("c:\\Go\\bin");
+
```

Comment:
Will the colon in `c:\...` possibly break things? It gets added for Unix platforms. Perhaps we should check check the current platform?